Benutzerdoku/Admin.tex

```diff
@@ -0,0 +1,85 @@
+
+
+
+\subsubsection{Einrichten}
+	\includegraphics[width=\textwidth]{Bilder/Admin/Setup}
+	\\
+	\\
+Die Anwendung muss von dem Administrator als ersten Schritt eingerichtet werden bevor diese von den Studenten genutzt werden kann. Dazu öffnet sich beim erstmaligen ausführen und auswählen des Button \textit{Administrator} ein Fenster. In das Textfeld \textit{Startdatum} wird der Beginn der Prüfungsphase eingetragen, wahlweise kann das Datum auch über das Kalender Icon neben dem Textfeld ausgewählt werden. In das Textfeld \textit{Anzahl der Gruppen} wird die Anzahl der Gruppen eingetragen die in der vorher festgelegten Prüfungsphase eine Präsentation halten. 
+\\
+\begin{wrapfigure}{R}{0.2\textwidth}
+	\centering
+	\includegraphics[width=0.20\textwidth]{Bilder/Buttons/Generieren}
+\end{wrapfigure}
+
+Durch betätigen des Buttons \textit{Generieren} erstellt die Anwendung eine Listenübersicht für vier Wochen mit dem eingetragenen Datum als Startdatum.
+
+
+
+\subsection{Tab}
+
+\begin{wrapfigure}{R}{0.2\textwidth}
+	\centering
+	\includegraphics[width=0.20\textwidth]{Bilder/Admin/Group-management-btn}
+\end{wrapfigure}
+Es stehen zwei Tabs zur Verfügung. Diese sind oben links in der Anwendung zu finden.
+
+
+\subsubsection{Gruppen}
+	\includegraphics[width=\textwidth]{Bilder/Admin/Group-management}
+	\\
+	\\	
+In den Tab Gruppen kann der Administrator mit dem Button \textit{Hinzufügen} eine neue Gruppe anlegen. Diese bekommt automatisch den Namen Group mit einer fortlaufenden Nummerierung.\\Mit dem Button \textit{Löschen} können bereits vorhandene Gruppen gelöscht werden auch wenn diese schon gebucht oder reserviert haben.
+
+\subsubsection{Termine}
+	\includegraphics[width=\textwidth]{Bilder/Admin/Date-management-Admin}
+	\\
+	\\
+In dem Tab Termine kann der Administrator die Listenübersicht der Wochen einsehen. Hier sind auch die Zustände der Tage eingetragen, sprich ob ein Tag
+\begin{itemize}
+	\item Aktiviert 
+	\item Deaktiviert 
+	\item Reserviert 
+	\item Gebucht
+\end{itemize}
+ist.
+\\ Mit dem Button \textit{Bearbeiten} können die Tage bearbeitet werden. Es öffnet sich für jeden Zustand ein entsprechendes Fenster.
+
+
+\paragraph{Aktiviert}
+\begin{wrapfigure}{R}{0.2\textwidth}
+	\centering
+	\includegraphics[width=0.4\textwidth]{Bilder/Buttons_clicked/Admin_Edit-Button-no-Booking}
+\end{wrapfigure}
+Ist der Tag aktiviert aber es sind keine Buchen oder Reservierungen vorhanden öffnet sich ein Fenster. In diesem kann der Administrator die Zeiträume festlegen in welchen eine möglich ist. 
+\\Mit dem Button \textit{Deaktivieren} kann der Tag deaktiviert werden. Dies ist immer möglich auch wenn eine Reservierung oder eine Buchung vorhanden ist.
+
+
+\paragraph{Deaktiviert}
+Ist ein Tag deaktiviert kann dieser mit klicken des Buttons \textit{Bearbeiten} und daraufhin mit betätigen des Buttons \textit{Aktivieren} wieder aktiv geschaltet werden.
```

Comment:
In dieser Ansicht können außerdem wieder Bemerkung und Zeitfenster bearbeitet werden